### DISCUSSION TITLE
Error: i.map is not a function

### DISCUSSION THREAD

**Discussion by sushantp053** [AntlerVC/firetable#975] [Support / Q&A] [answered]
👉 **Please describe the steps that you took that led to this bug.**
This error gets when I want to create the same project again with the same name. 

I created a project with "servicescrm" but due to some reason, I aborted the process.

again after some time, I started creating projects with the same name. Then error is thrown.

**Comment by RajGM** [comment]
I would suggest deleting the firebase and creating it again.

**Reply by eleon13** [reply]
This happened to me too. I left a screenshot below. 
I just made a new Rowy account and when I go to create a new project I get this error. I have authenticated my Google 
![Screenshot 2022-11-27 at 7 49 30 PM](https://user-images.githubusercontent.com/119269122/204169998-6605131f-2a82-42fb-a882-0325d83834f8.png)
account too.

**Comment by BeeBombshell** [comment, marked as answer]
Hey @sushantp053 and @eleon13, does this issue persist, or has this been resolved?

**Reply by BeeBombshell** [reply]
Hey guys, closing this discussion since it has been unattended for a long time. @sushantp053, @eleon13, if you're still facing this issue, feel free to create a new one, or [book a 1:1 chat](https://rowy.io/meet) with us so that we can help you resolve this.
